### DISCUSSION TITLE
Failed to initialize workflow glsp server

### DISCUSSION THREAD

**Discussion by wss29** [eclipse-glsp/glsp#836] [Q&A]
hi,
   I download glsp-theia-integration and follow the document steps
yarn install
yarn download:exampleServer
cd examples/browser-app
yarn start
when I load localhost:3000, I get an error, how to deal with it? 
![image](https://user-images.githubusercontent.com/4537809/209267032-a6486549-0beb-4ead-870b-1c2f992ed511.png)

**Comment by tortmayr** [comment]
Hi @Pagina,
unfortunately I cannot reproduce this issue. If I follow your steps in a clean repository everything works as expected.

Maybe something went wrong during the download? Could you please check if the server jar exists (`glsp-theia-integration/examples/workflow-theia/server/org.eclipse.glsp.example.workflow-1.1.0-SNAPSHOT-glsp.jar`)?
Do you see any errors in the console when executing `yarn start`?

**Comment by rsoika** [comment]
@Pagina I would guess that your server is simply not running. As @tortmayr mentioned you should see some kind of errors in the console? You should see a server message like this:

```
[main] INFO  SocketGLSPServerLauncher - The GLSP server is ready to accept new client requests on port: 5007
[GLSP-Server]:Startup completed
```

**Comment by wss29** [comment]
Hi,
   I find the problem, glsp-theia-integration need "node": ">=16.11.0"， My office computer use node14， so this afternoon I download node18 and install it. and I get the problem I post. just now I come home using my home computer which has node16, I follow all steps in my office, and the graph can display normally, I am very surprised, Is it the node version problem? So I uninstall node16 and install node18, and I get the same problem I post upside. I think about it and find node18 installed python 3.11.0, and node16 installed python 3.10.0. Even I uninstall node16, python 3.10.0 is still on my computer. so I uninstall python 3.10.0 and run again everything is ok.
The problem is caused by two different versions of python installed on the same computer. Next Monday I will go to the office to ensure the situation in my office computer

**Comment by tortmayr** [comment]
@Pagina You are right. I tested it with node 18 as well and it is not working. I have created #837 to track this.

**Comment by rileydanejohnston** [comment]
Hello,
I encounter this issue every time I launch the project. I can solve the problem by clicking the restart button on the client/extension host in the "Run and Debug: Call Stack" section. I'd like to have the project start without interruption but it appears I'm missing something. I'm using the workflow example for my project, with a node server and vscode client.

I get the messages @rsoika describes when I look at the workflow server:
![Screenshot 2023-01-12 at 10 32 20 AM](https://user-images.githubusercontent.com/54558543/212125359-8ada5692-3393-4c99-8f4b-f5fdf661c64a.png)

This is the error I get when I go to the client/extension host:
![Screenshot 2023-01-12 at 10 26 44 AM](https://user-images.githubusercontent.com/54558543/212124095-162f1ba7-eafc-4d0f-94aa-bd6cfa1dc2b8.png)

**Reply by rileydanejohnston** [reply]
I think I had a few configuration issues because the problem appears to be solved. It turns out my server was in watch-mode so there was no compiled files directory (yet my changes were being reflected in the graph). I also added the outFiles path in my launch.json file. It seems to work now!
